syntax = "proto3";

package mprpc;

message RpcHeader
{
    bytes service_name = 1;         //服务名
    bytes methon_name = 2;          //方法名
    uint32 args_size = 3;           //参数大小
}